### ISSUE TITLE
fails to connect to mysql and postgresql

### ISSUE THREAD

**Issue by brancomat** [ARPA-SIMC/provami#27]
in the past we've successfully tested `provami-qt` with sqlite:
 ``` $  provami-qt sqlite://nomefile```
and odbc:
 ``` $ provami-qt odbc://test@test```

since we dropped odbc support we couldn't find a syntax to make it work with mysql and postgresql db:
```
$ provami-qt mysql://radicchio:3306/sapone?user=xxxx&password=xxxx
terminate called after throwing an instance of 'wreport::error_notfound'
  what():  invalid parameter "mysql://radicchio:3306/sapone?user"
```

```
$ provami-qt postgresql://test:test@radicchio/qc_test_pgsql
terminate called after throwing an instance of 'QUnhandledException'
  what():  std::exception
```
For testing purposes the same db we agreed upon for dballe deploy are available (`test_build` on radicchio's mysql and posgresql)

Right now it's unclear if provami-qt supports any mysql/postgresql syntax, IMHO `provami-qt` should also:
- [x] when launched with the `-h --help` option, show sqlite, mysql and postgresql syntax 
- [x] (optional but recommended)  use the same syntax of other dballe tools like `dbadb`

**Comment by spanezz** [comment]
The URL syntax should be the same as this one, can you confirm?
https://github.com/ARPA-SIMC/dballe/blob/master/doc/fapi_connect.md

**Comment by pat1** [comment]
for me yes

**Comment by brancomat** [comment]
"Should be" as in "wouldn't it be great if it was that one": yes!
"Should be" as in "it should alrealdy be like this": unfortunately no (see output pasted above)

(I can file a different issue for mysql and postgres not working, I just wanted to be sure of the correct syntax to begin with)

**Comment by spanezz** [comment]
The syntax is confirmed, I changed the title of this issue

**Comment by pat1** [comment]
[this is Enrico from Paolo's computer] It should be fixed in b398e1ba. It looks like I pushed it but didn't close the bug. Can you confirm that it works on current master?

**Comment by cmarsigli** [comment]
Hello!
Any news about using provami with postgres?
Chiara

**Comment by brancomat** [comment]
> Any news about using provami with postgres?

this issue is currently blocked by issue #28

**Comment by brancomat** [comment]
Fixed in provami v1.4-2
